ml-metadata-server/rockcraft.yaml

```diff
@@ -0,0 +1,58 @@
+# Based on: https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol/Dockerfile
+name: ml-metadata-server
+summary: Part of the ML Metadata (MLMD) framework.
+description: |
+  ML Metadata is a framework for managing and tracking metadata associated with ML workflows. 
+  It is part of the TensorFlow Extended (TFX) ecosystem, but it can be used independently as well.
+version: "1.14.0"
+license: Apache-2.0
+base: ubuntu@22.04
+run-user: _daemon_
+
+platforms:
+  amd64:
+
+parts:
+  server:
+    plugin: nil
+    source: https://github.com/google/ml-metadata.git
+    source-tag: v1.14.0
+    build-packages:
+      - build-essential
+      - git
+      - clang
+      - cmake
+      - make
+      - ca-certificates
+      - musl-dev
+      - openssl
+      - curl
+      - unzip
+      - software-properties-common
+      - python3-distutils
+      - python-is-python3
+      - python3-dev
+    build-environment:
+      - CGO_ENABLED: 0
+      - GOOS: linux
+      - BAZEL_VERSION: 5.3.0
+    override-build: |
+      # Set up Bazel 5.3.0
+      mkdir /bazel && cd /bazel
+      curl -H "User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36" -fSsL -O https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh && \
+      curl -H "User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36" -fSsL -o /bazel/LICENSE.txt https://raw.githubusercontent.com/bazelbuild/bazel/master/LICENSE && \
+      chmod +x bazel-*.sh && ./bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh && \
+      cd $CRAFT_PART_BUILD && rm -f /bazel/bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh
+      
+      #Build
+      bazel build -c opt --action_env=PATH \
+      --define=grpc_no_ares=true --verbose_failures \
+      //ml_metadata/metadata_store:metadata_store_server --cxxopt="-std=c++17"
+
```

Comment:
missing the step to [copy libmysqlclient source onto third_party folder](https://github.com/google/ml-metadata/blob/v1.14.0/ml_metadata/tools/docker_server/Dockerfile#L58-L60)